src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportFAT2SectorStandardToPL.sql

```diff
@@ -79,7 +79,12 @@ BEGIN TRY
 								,[TypicalDuration]
 								,[MaxFunding]
 								,[IsActive]
-								,[VersionApprovedForDelivery]
+								,[VersionApprovedForDelivery]	
+								,[EffectiveFrom]
```

Comment:
I thought we discussed we will remove EffectiveFrom and EffectiveTo Fields?

Reply:
EffectiveFrom and EffectiveEndDates are different VersionLatestStart and EndDate , so EffectiveFrom and EffectiveEndDates are added back to schema